src/main/java/org/example/bookstore/service/impl/BookServiceImpl.java

```diff
@@ -47,6 +48,8 @@ public BookDto update(Long id, CreateBookRequestDto requestDto) {
     @Override
     public void deleteById(Long id) {
         checkIfBookExists(id);
```

Comment:
You can also add the same checking for ISBN in create method (what if I try to create a new book with existing ISBN)